### PR TITLE
Fix percent being non-inclusive

### LINKED ISSUE #145: Percent in IPL is non-inclusive
Pallet IPL uses Percent for execution threshold and voting weights, but Percent describes the range between 0 and 1 non-inclusive, meaning it's not possible to set those fields to 1 as the max for Percent is 99.

### PR DESCRIPTION
**Pull request by S0raWasTaken**
closes #145

### CODE REVIEW

**Comment by S0raWasTaken** [comment]
I'll test this in the node before marking it as ready for review

File: primitives/src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,12 @@ use codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
 use sp_runtime::Percent;
 
+#[derive(Encode, Decode, Clone, Copy, Eq, PartialEq, MaxEncodedLen, Debug, TypeInfo)]
+pub enum Percentage {
```
Comment (arrudagates) [inline primitives/src/lib.rs:8]: Percentage is too vague for this context, it's difficult to tell apart from Percent just by the name.

```suggestion
pub enum OneOrPercent {
```

**Review by arrudagates — CHANGES_REQUESTED** [review]

File: primitives/src/lib.rs (hunk 1 above)
Comment (S0raWasTaken) [reply primitives/src/lib.rs]: Resolved in [a8c5e0e](https://github.com/InvArch/InvArch-Frames/pull/147/commits/a8c5e0e90b6cb3d4ffd7e21943ba688e4c3cdb6a)

**Review by arrudagates — APPROVED** [review]

**Comment by arrudagates** [comment]
@S0raWasTaken merge when you're ready

**Pull request merged**